src/bot.py

```diff
@@ -42,7 +42,7 @@
 
 ---
 
-[^(What's this?)](https://www.reddit.com/r/minecraftmemes) ^| [^(Source Code)](https://github.com/Xeoth/GoldenNuggetBot) ^(| Made by Xeoth, coderDude69 and LPDtheGamer)
+[^(r/MinecraftMemes Nether Event)](https://www.reddit.com/r/minecraftmemes/"LINK_TO_PIE'S POST") ^| [^(About this bot)](https://www.reddit.com/user/GoldenNugBot/comments/hdzqwi/confused_read_this/) ^(| Made by Xeoth, coderDude69 and LPDtheGamer)
```

Comment:
You can do `https://reddit.com/r/minecraftmemes/about/sticky` and it'll redirect to the top sticky 👀 

Wanna do that or should I just merge as is?

Reply:
Now fixed, that's a pretty cool feature tbh